README.md

```diff
@@ -67,6 +68,9 @@ If you are not a programmer, but would like to contribute, check out the [Awesom
 
 - [LightTable](https://github.com/LightTable/LightTable/labels/good%20first%20issue) _(label: good first issue)_ <br>  The Next Generation code editor! One of the top funded projects on KickStarter.
 
+## CSS
+- [sButtons](https://github.com/shahednasser/sbuttons/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) _(label: good first issue)_ <br> Simple buttons you can easily use for your next project.
```

Comment:
Hi @shahednasser 

Would you be able to change the URL to link directly to the label page (see other entries in this list).
Will be great it you could also add a new line between category name and the first item.

Thanks!